fix: BROS-661: sync SDK

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/20143027955)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/695
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20138148098

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-BROS-661-sync-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20138148098

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20142930853)
> Successfully merged:  1 file changed, 1 deletion(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-BROS-661-sync-sdk'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20142930853

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20142941112

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20143027955

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/20144855249
